src/app/components/search/SearchResultsCards/FactCheckCard.js

```diff
@@ -22,7 +22,7 @@ const FactCheckCard = ({
   <div className={`${styles.factCheckCard} fact-check-card`}>
     <Card
       footer={
-        <Tooltip title={teamName}>
+        <Tooltip arrow title={teamName}>
```

Comment:
not really thematically part of this PR, but just adding this to match the other tooltips on this page